app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -241,8 +262,8 @@ public static int nthToLastCharIndex(int elementNumber, String str, char element
      * Formats input to plain mm:ss format
      */
     public static String formatTimer(long timer) {
-        final long min = TimeUnit.SECONDS.toMinutes(timer);
-        final long sec = TimeUnit.SECONDS.toSeconds(timer - TimeUnit.MINUTES.toMillis(min));
-        return String.format("%02d:%02d", min, sec);
+        final long min = timer / 60;
```

Comment:
You seem to be doing exactly what was being done before, but in a more 'illegible' way. See ` TimeUnit.SECONDS.toMinutes(timer);`:

```
public long toMinutes(long d) { return d/(C4/C3); }// C4/C3 == 60L
```